<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Transaction</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: rgb(19, 27, 38);
}</string>
  </property>
  <widget class="QLabel" name="updateDescLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;New Description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="updateDescInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter a description...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateSubmitButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #3b82f6;
color: #ffffff;
border: none;
border-radius: 8px;
padding: 5px 10px;
font-weight: 600;
}

QPushButton:hover {
background-color: #2563eb;
}

QPushButton:pressed {
background-color: #1d4ed8;
}</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateCancelButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #1e3a8a;
color: #93c5fd;
border: 1px solid #3b82f6;
border-radius: 8px;
padding: 5px 10px;
}

QPushButton:hover {
background-color: #1e40af;
color: #ffffff;
border-color: #60a5fa;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="basicDetails">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Basic Details</string>
   </property>
   <widget class="QLabel" name="updateNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;New Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="updateNameInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter name...</string>
    </property>
   </widget>
   <widget class="QLabel" name="updateAmountLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;New Amount&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="updateAmountInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="updateDateLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;New Date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="updateDateInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="dropDowns">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Type and Category</string>
   </property>
   <widget class="QComboBox" name="updateCategoryInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="updateTypeInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Expense</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Income</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="updateCaregoryLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;New Category&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="updateTypeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;New Type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <zorder>dropDowns</zorder>
  <zorder>basicDetails</zorder>
  <zorder>updateDescLabel</zorder>
  <zorder>updateDescInput</zorder>
  <zorder>updateSubmitButton</zorder>
  <zorder>updateCancelButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
